views/hooks/useChartsList.jsx

```diff
@@ -69,16 +69,18 @@ export default function useChartsList() {
       setNotification({ open: true, message: error.message })
     }
   }
-  const loadCharts = async () => {
+  const loadCharts = async (queryParams = {}) => {
```

Comment:
do we need to default this or can we leave it as undefined if there are no query params?

Reply:
we don't have to default it 
[216b66e](https://github.com/AMP-SCZ/dpdash/pull/560/commits/216b66e967a28fae64661657855e58bd0984fef3)